app/templates/render-tree.hbs

```diff
@@ -1,5 +1,5 @@
 {{#unless isClosed}}
-  {{warning-message storage=storage}}
+  {{warning-message closed=(action "setIsClosed")}}
```

Comment:
If you bind `setIsClosed=(action "setIsClosed")` you shouldn't need the action inside the component.